recipes/evm/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set version = "2.1.0" %}
+{% set name = "evm" %}
+{% set sha256 = "3ee69ad822a35204ff02b670fe047d8b0da250cf9f43ebabbe33ce47f498e668" %}
+
+package:
+  name: "{{ name }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://github.com/EVidenceModeler/EVidenceModeler/releases/tag/EVidenceModeler-v{{ version }}.tar.gz"
+  sha256: "{{ sha256 }}"
+
+build:
+  number: 0
+  script: "make"
```

Comment:
I guess you will need to copy some binaries to $PREFIX/bin.
See other recipes' `build.sh` for inspiration!